core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -1058,6 +1058,57 @@ public void testJoinUsingDynamicTable() {
     sql("select * from dept, lateral table(ramp(dept.deptno))").ok();
   }
 
+  @Test public void testTableFunctionInSelect() {
+    sql("select table_func(dept.deptno) as (f0, f1) from dept")
+        .conformance(SqlConformance.HIVE)
+        .ok();
+  }
```

Comment:
What will happen if there is a table function without as ? Can you add a test about it?

Reply:
It works well without `as`